dashboard/api/pkg/store/migrations/0003_data_moderation.up.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TYPE moderation_type ADD VALUE IF NOT EXISTS 'result';
+
+-- A result moderation request is attached to a job moderation request; one
+-- result moderation for each result storage spec in the job.
+CREATE TABLE result_moderation_request_extra (
```

Comment:
Are these only ever created when a job_moderation_request is, or should it have it's own creation timestamp?